<launch>

<param name="/wheelchair_robot/param/rostolabelimg/image_topic" type="string" value="/zed/zed_node/left/image_rect_color" />
<param name="/wheelchair_robot/param/rostolabelimg/bag_location" type="string" value="/home/tomos/ros/wheelchair/catkin_ws/src/wheelchair_dump/dump/bags/env1-run1_2021-08-05-14-32-31.bag" />
<param name="/wheelchair_robot/param/rostolabelimg/img_location" type="string" value="/home/tomos/ros/wheelchair/catkin_ws/src/wheelchair_dump/dump/rostolabelimg/img/" />
<param name="/wheelchair_robot/param/rostolabelimg/xml_location" type="string" value="/home/tomos/ros/wheelchair/catkin_ws/src/wheelchair_dump/dump/rostolabelimg/xml/" />
<param name="/wheelchair_robot/param/rostolabelimg/confidence_threshold" type="double" value="0.5" />

<node pkg="rostolabelimg" name="bag2label" type="bag2label.py" output="screen"></node>
<!--<node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
<remap from="image" to="/wheelchair_robot/mobilenet/annotated_camera" />
</node>-->
</launch>
